mastracode/src/web/ui/styles.css

```
@@ -2099,6 +2149,75 @@ body::before {
+  font-weight: 500;
+  cursor: pointer;
+  transition:
+    background 0.12s,
+    color 0.12s,
+    border-color 0.12s;
+}
+.sidebar-github-btn:hover {
+  background: var(--bg-surface2);
+  color: var(--fg);
+  border-color: var(--accent);
+}
```

Stylelint (17.13.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)